routing/disableInternet.yaml

```diff
@@ -7,17 +7,6 @@
     - ../defaults.yaml
     - ../config.yaml
     - ../config.yml
-  handlers:
```

Comment:
This is weird to me, you are now not telling the kernel to stop/start accepting packets not intended for 'this' host. I heavily suspect that this was left on during local testing and that dropping iptables rules then appears to be sufficient.

Reply:
Docker uses ipv4 forwarding to route packets to containers. If disabled, containers will als not have access to the internet.